Resources/Prototypes/Catalog/Fills/Boxes/general.yml

```diff
@@ -275,24 +275,24 @@
       - DroneUsable
 
 - type: entity
-  name: syndicate EVA box
-  parent: BoxCardboard
-  id: BoxSyndicateEVA
-  description: A box containing one Syndicate approved EVA suit.
-  components:
-  - type: Item
-    size: 40
-  - type: Storage
-    capacity: 40
-    size: 10
-    whitelist:
-      components:
-      - Clothing
-  - type: StorageFill
-    contents:
-      - id: ClothingHeadHelmetSyndicate
-      - id: ClothingOuterHardsuitSyndicate
-  - type: Sprite
-    layers:
-      - state: box_of_doom
-      - state: writing
+   name: syndicate EVA box
+   parent: BoxCardboard
+   id: BoxSyndicateEVA
+   description: A box containing one Syndicate approved EVA suit.
+   components:
+   - type: Item
+     size: 80
+   - type: Storage
+     capacity: 80
+     size: 10
+     whitelist:
+       components:
+       - Clothing
+   - type: StorageFill
+     contents:
+       - id: ClothingHeadHelmetSyndicate
+       - id: ClothingOuterHardsuitSyndicate
+   - type: Sprite
+     layers:
+       - state: box_of_doom
+       - state: writing
```

Comment:
Sorry for the spam, never did reviews before.
*ahem*
Test test.

Reply:
Ok, this is better. Still not 100%, but I won't do more just so I don't implode your inbox.
The following comment

> There's an extra space in front of all lines here. Please remove the spaces, I'm 99% this breaks the linter and fails to load (tested locally)

was meant for this file.

Reply:
Well spotted, thanks. Mobile error ;)